documentation/src/main/asciidoc/topics/yaml/cache_template_inheritance.yaml

```diff
@@ -1,13 +1,15 @@
 infinispan:
   cacheContainer:
-    - distributedCacheConfiguration:
-      name: "base-template"
-      encoding:
-        mediaType: "application/x-protostream"
-      expiration:
-        lifespan: "5000"
-        maxIdle: "1000"
-    - distributedCacheConfiguration:
-      name: "extended-template"
-      expiration:
-        lifespan: "10000"
+    caches:
+      base-template:
+        distributedCacheConfiguration:
+          mode: "SYNC"
+          encoding:
+            mediaType: "application/x-protostream"
+          expiration:
+            lifespan: "5000"
+            maxIdle: "1000"
+      extended-template:
```

Comment:
the `extended-template` needs to extend `base-template`

Reply:
thanks @pruivo I've switched things around in the example configs so it looks like the extended template actually "extends" the base template. also, in the yaml, I did notice that the extended-template didn't reference the base-template correctly, which I've fixed.

I've also started the server with these examples double check them.